### PR TITLE
Update owners file.

### PR DESCRIPTION
**Pull request by HumairAK**
As per discussion [here](https://github.com/open-infrastructure-labs/ops-issues/issues/16).
The file is the same as the one [here](https://github.com/operate-first/apps/blob/master/OWNERS).

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 approvers:
-  - anishasthana
-  - hemajv
+  - 4n4nd
   - HumairAK
   - tumido
-  - martinpovolny
-  - 4n4nd
-  - durandom
+  - hemajv
```
Comment (anishasthana) [inline OWNERS:7]: Shouldn't hema _not_ be in this list?

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (HumairAK) [reply OWNERS]: updated

**Review by tumido — APPROVED** [review]
:ship:

**Comment by tumido** [comment]
@anishasthana can you please re-review this one. Seems like GH decided you want some changes...

image

**Review by anishasthana — APPROVED** [review]
Heh, I had requested some changes late last night. Approving now :-)

**Pull request merged**